onyo/lib/onyo.py

```diff
@@ -1081,6 +1119,12 @@ def set(self, paths: Iterable[Union[Path, str]], values: dict, dryrun: bool,
         if diff and dryrun:
             self.restore()
 
+        self.clear_caches(assets=True,  # `onyo set` can rename assets
```

Comment:
Same comments as above.

Not a fan of the duplication of this; it will require that they remain in sync. But it is unclear to me why the caches are cleared explicitly above.